system.go

```diff
@@ -1,23 +1,68 @@
 package utils
 
 import (
+	"fmt"
 	"time"
 
 	log "github.com/Sirupsen/logrus"
 )
 
 // StopEtcd stops etcd on a specific host
-func StopEtcd(node TestbedNode) error {
-	log.Infof("Stopping etcd")
-	return node.RunCommand("pkill etcd && rm -rf /tmp/etcd")
+func StopEtcd(nodes []TestbedNode) error {
+	for _, node := range nodes {
+		log.Infof("Stopping etcd on node %s", node.GetName())
+
+		if err := node.RunCommand("sudo systemctl stop etcd; rm -rf /var/lib/etcd"); err != nil {
```

Comment:
I think the `ExecStopPost` action in the systemd unit takes care of removing the data-dir so we need not explicitly do it here.